src/lidar_measurement_model_beam.cpp

```diff
@@ -87,6 +87,10 @@ void LidarMeasurementModelBeam::loadConfig(
   double ang_total_ref;
   pnh.param("ang_total_ref", ang_total_ref, M_PI / 6.0);
   sin_total_ref_ = sinf(ang_total_ref);
+
+  int use_label_leq;
+  pnh.param("use_label_leq", use_label_leq, static_cast<int>(0xFFFFFFFF));
```

Comment:
is this parameter name correct? It sounds like a boolean value, but actually it is used as a threshold.

Reply:
`filter_label_if_leq` or `filter_label_max`? What do you think?

Reply:
`filter_label_max` or `filter_label_leq`?